apps/sync/test/unit/sync-engine/kinto-mock.js

```diff
@@ -8,6 +8,7 @@
 
 var Kinto = (function() {
   var mockProblem =  null;
+  var listData = {};
```

Comment:
I was previously keeping this data inside the mocked Collection object, but now keeping it globally here, which is a slightly more realistic simulation of how Kinto.js persists data in IndexedDB.